contracts/KyberPayWrapper.sol

```diff
@@ -0,0 +1,181 @@
+pragma solidity 0.4.18;
+
+
+import "./ERC20Interface.sol";
+import "./Withdrawable.sol";
+import "./ReentrancyGuard.sol";
+
+
+interface KyberNetwork {
+    function tradeWithHint(
+        ERC20 src,
+        uint srcAmount,
+        ERC20 dest,
+        address destAddress,
+        uint maxDestAmount,
+        uint minConversionRate,
+        address walletId,
+        bytes hint)
+    external
+    payable
+    returns(uint);
+}
+
+
+contract KyberPayWrapper is Withdrawable, ReentrancyGuard {
+    ERC20 constant public ETH_TOKEN_ADDRESS = ERC20(0x00eeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeee);
```

Comment:
I think this constant is already defined in `Withdrawable`

Reply:
as we saw it is not.